src/BTVNanoCommissioning/workflows/ctag_DY_valid_sf.py

```diff
@@ -156,7 +170,7 @@ def process(self, events):
         sjets = event_jet[event_level]
         njet = ak.count(sjets.pt, axis=1)
         # Find the PFCands associate with selected jets. Search from jetindex->JetPFCands->PFCand
-        if self._campaign != "Rereco17_94X":
+        if "PFCands" in events.fields:
```

Comment:
as mentioned by @AnnikaStein in `correction.py`, `if <STRING> in` should be avoided, as a non-0empty string is always returning `True`

Reply:
thx for clarifying! saves a lot of work then

Reply:
I think `if <STRING> in` is fine, but never use `if <string>` as it would be always `True`. But specify whether string in a list of strings should be correct way